Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+
+- [Upgrading projects](#upgrading-projects-to-200)
+- [What's new](#whats-new-in-200)
+- [Known issues](#known-issues-in-200)
+
+This release of the Microsoft Mixed Reality Toolkit 2.0.0 supports the following devices and platforms.
+
+- Microsoft HoloLens 2
+- Microsoft HoloLens (1st gen)
+- Windows Mixed Reality Immersive headsets
+- OpenVR
+
+The following software is required.
+
+- Microsoft Visual Studio (2017 or 2019) Community Edition or higher
+- Windows 10 SDK 18362 or later (installed by the Visual Studio Installer)
+- Unity 2018.4, 2019.1 or 2019.2
+
+### Upgrading projects to 2.0.0
+
+Since the RC2 release, there have been several changes that may impact your application projects,
+including some files moving to new folder locations. For the smoothest upgrade path in your
+projects, please use the following steps.
+
+1. Close Unity
+1. Delete all **MixedRealityToolkit** (you may not have all listed folders)
+    - MixedRealityToolkit
+    - MixedRealityToolkit.Examples
+    - MixedRealityToolkit.Extensions
+    - MixedRealityToolkit.Providers
+    - MixedRealityToolkit.SDK
+    - MixedRealityToolkit.Services
+    - MixedRealityToolkit.Tools
+1. Delete your **Library** folder
+1. Re-open your project in Unity
+1. Import the new unity packages
+    - Foundation - _Import this package first_
+    - (Optional) Tools
+    - (Optional) Extensions,
+    - (Optional) Examples are optional)
+1. For each scene in your project
+    - Delete **MixedRealityToolkit** and **MixedRealityPlayspace**, if present, from the Hierarchy
+    - Select **MixedRealityToolkit -> Add to Scene and Configure**
+
+> [!Important]
+> Some profiles have been changed (properties have been added) in 2.0.0. If you have created custom
+profiles, please open them to verify that all of the updated properties are correctly configured.
+
+### What's new in 2.0.0
+
+**BoundingBox**
+
+- Improvements
+    - Normalized the handle asset size and scaling logic
+    - The handle asset is now sized to 1-meter
+    - Default values and examples are updated
+- New features
+    - Animated Handle by Proximity
+    - Match the HoloLens 2 shell behavior
+    - You can now make the handles appear only when your hand is close to them
+- New example scene
+    - The BoundingBoxExample scene shows various types of configurations
+
+> [!Important] 
+> Normalized asset handle size and scaling logic require updated handle scale values. With this change, the existing bounding box handle will be displayed in a very large size. Please refer to the **Bounding box handle styles** section in the [Bounding Box](README_BoundingBox.md) documentation for updated recommended values for the handle size. You can also find handle configuration examples in the **BoundingBoxExamples** scene. 
+<img src="../Documentation/Images/BoundingBox/MRTK_BoundingBox_HandleStyles2.png" width="450">
+
+**Clipping Example**
+
+The ClippingExamples scene, demonstates using the MRTK Standard Shader’s new clipping feature.
```

Comment:
>ClippingExamples scene [](start = 4, length = 22)

Consider adding a link to the clipping examples scene.

Reply:
added